packages/scene-composer/src/components/toolbars/common/ItemContainer.spec.tsx

```diff
@@ -53,6 +53,13 @@ describe('ItemContainer', () => {
     setFeatureConfig({});
   });
 
+  it('should render selected item properly with custom menuHeight', () => {
+    const { container } = render(
+      <ItemContainer item={{ ...baseItem, isSelected: true }} type='action-select' menuHeight='100px' />,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
```

Comment:
Non-blocker: Consider how to test this without using a snapshot. Please fix this forward.

Reply:
done